NitroxPatcher/Patches/Dynamic/CyclopsSonarButton_Update_Patch.cs

```diff
@@ -16,8 +16,8 @@ public class CyclopsSonarButton_Update_Patch : NitroxPatch, IDynamicPatch
     {
         public static readonly MethodInfo TARGET_METHOD = Reflect.Method((CyclopsSonarButton t) => t.Update());
 
-        private static readonly OpCode INJECTION_OPCODE = OpCodes.Ldsfld;
-        private static readonly object INJECTION_OPERAND = Reflect.Field(() => Player.main);
+        public static readonly OpCode INJECTION_OPCODE = OpCodes.Ldsfld;
+        public static readonly object INJECTION_OPERAND = Reflect.Field(() => Player.main);
```

Comment:
`internal` is preferred. The Nitrox.Test project has exclusive access to all `internal` members for this reason.